rocAL/rocAL/source/rali_api_data_transfer.cpp

```diff
@@ -111,3 +111,15 @@ raliCopyToOutput(
     return RALI_OK;
 }
 
+void
```

Comment:
Please change the function returning RaliStatus

Reply:
Sure, will make this change in the next version